SECURITY.md

```diff
@@ -0,0 +1 @@
+See https://github.com/lima-vm/lima/blob/master/SECURITY.md
```

Comment:
I understand the inherent logic of pointing to a central `SECURITY.md` file, but since that file is just redirecting you to the `MAINTAINERS.md` file, wouldn't it be better to copy that file instead of having a double indirection?

```suggestion
# Reporting security issues

To report security issues, please reach out to the [maintainers](https://github.com/lima-vm/lima/blob/master/MAINTAINERS.md) directly via an e-mail.
```

Reply:
That may make it hard to update the security policy